### PR TITLE
create ApplicationForm and FormSubmittedMessage components

### PR DESCRIPTION
**Pull request by prabshah**
=> created new folder called 'components' under src.
=> created two components 'ApplicationForm' and 'ForSubmittedMessage'.
=> added CSS an jQuery CDN to index.html page.
=> commented out the default styling of App component.
=> inported newly created components to App.js and render for testing purpose.

### CODE REVIEW

File: frontend/src/components/ApplicationForm.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+	constructor() {
+		super();
+	}
+
+	render() {
+		return (
+			<div className="application-form container">
+				<h1 className="text-center mt-5 pt-4 pb-3 display-5">Student Application Form</h1>
+				<form className="mb-4 mt-5">
+					<div className="form-group  mt-5">
```
Comment (OlenaKashuba) [inline frontend/src/components/ApplicationForm.js]: You have a lot of repeating components (input + label within a div), i think it is a good sign to make it a separate component and pass type, name, id etc. as props

File: frontend/public/index.html [hunk 2]
```diff
@@ -10,6 +10,10 @@
     -->
```
Comment (OlenaKashuba) [inline frontend/public/index.html]: Great explanation with pull request, so clear what you have done!

File: frontend/src/components/ApplicationForm.js (hunk 1 above)
Comment (prabshah) [reply frontend/src/components/ApplicationForm.js]: Yes, the code can be improved by splitting the form into smaller components but I wanted to create a form just with HTML and not using Props or State for now. As Gerald said he wants to see pure HTML on our components. We can change it later once our pages start to be more interactive.

File: frontend/src/components/FormSubmittedMessage.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+				width="150"
+				className="img-fluid mt-5 pt-4"
+			/>
+			<h1 className="mt-5 pb-3 display-5">Thank you!</h1>
+			<h3 className="pt-4 pb-3 display-5">Your detail have been submitted sucessfully.</h3>
+		</div>
+	);
+};
+
+export default FormSubmittedMessage;
```
Comment (LouiaR) [inline frontend/src/components/FormSubmittedMessage.js]: Great job :+1: @prabshah  PR well explained

**Comment by dahfool** [comment]
@prabshah Good progress

**Comment by dahfool** [comment]
@prabshah I have just merged @OlenaKashuba PR, which created a conflict, please fix.

**Pull request merged**